source/geometry/Quaternion.ooc

```diff
@@ -28,15 +29,9 @@ Quaternion: cover {
 	y ::= this imaginary y
 	z ::= this imaginary z
 
-	// NOTE: The coordinates are represented differently in C# Kean:
-	// x = this w
-	// y = this x
-	// z = this y
-	// w = this z
-
 	isValid ::= this real isNumber && this imaginary isValid
 	isIdentity ::= this real equals(1.f) && this imaginary x equals(0.f) && this imaginary y equals(0.f) && this imaginary z equals(0.f)
-	isNull ::= this real equals(0.f) && this imaginary x equals(0.f) && this imaginary y equals(0.f) && this imaginary z equals(0.f)
+	isZero ::= this real equals(0.f) && this imaginary x equals(0.f) && this imaginary y equals(0.f) && this imaginary z equals(0.f)
```

Comment:
This could be mistaken for a quaternion representing zero rotation which is the same as `isIdentity`. Do we even use this property? I don't think we should ever create a (0,0,0,0) quaternion. Right now we are doing so in the FloatRotation3D default constructor, but I would probably remap that to identity later.